habitat/core/embodied_task.py

```diff
@@ -131,6 +135,43 @@ def __init__(
         self._sim = sim
         self._dataset = dataset
 
+        self._possible_actions = OrderedDict()
+        for action in config.POSSIBLE_ACTIONS:
+            assert action in registry.mapping["task_action"]
+            self._possible_actions[action] = registry.mapping["task_action"][
+                action
+            ]
```

Comment:
Maybe worth adding something like
```python
for i, v in enumerate(self._possible_actions.value()):
    self._possible_actions[i] = v
```

for cleaner lookup by index later?

Reply:
@erikwijmans, make sense, but then it tricky to check number of possible actions. Maybe worth to have separate dict with indexes.